lib/takeaway.rb

```diff
@@ -0,0 +1,80 @@
+require_relative 'order'
+require_relative 'menu'
+require_relative 'twilio'
+
+class Takeaway
+
+  def initialize(order = Order.new, text_client = TwilioClient.new)
+    @order = order
+    @order_done = false
+    @text_client = text_client
+  end
+
```

Comment:
User interface created from using the 'take_order' method is very nicely presented and easy to follow